docs/usage.md

```diff
@@ -62,10 +62,21 @@ the cloned repository.
 evg-module-manager disable --module wiredtiger
 ```
 
+## Performing git commit all
+
+Use the `git-commit` command to commit all your git tracked changes to your base repository and any
+modules that are currently enabled. The tool has already specified `--all` and `--message` option to git, which means
+it would automatically add and commit all the modifications that git has tracked.
+
+To commit all the tracked changes in base repo and any enabled modules:
+```bash
+evg-module-manager git-commit --commit-message my-commit
```

Comment:
I think it would be good if this example used a more realistic commit message, maybe something like:

```bash
--commit-message "my commit message"
```